src/components/attendee/Input/Input.module.scss

```diff
@@ -3,6 +3,8 @@
 @use "scss/attendee/theme";
 
 .input {
+  border: none;
```

Comment:
This is a tough call... Don't see the design ATM, but I'd guess not everywhere a border-less input is desired. At least this one should be passed as a prop to the `<Input />` component, something like a variance prop

```tsx
  border?:  "none" | "normal";

  const inputClassNames = classNames(CN.inputField, CN[border])
```

or a literal `variant` named one:

```tsx
  variant?: "profile" | "normal";

  const containerClassNames = classNames(CN.input, CN[variant]);
```

Reply:
added a `border` / `borderless` variations [7f5a37b](https://github.com/sparkletown/sparkle/pull/2840/commits/7f5a37baa83617db540f69b073ae690c37f12ce6)